pom.xml

```diff
@@ -79,6 +79,18 @@
             <id>mbuchwald</id>
             <name>Martin Buchwald</name>
         </developer>
+        <developer>
+            <id>alegmarra</id>
+            <name>Alejandro Garcia Marra</name>
+        </developer>
+        <developer>
+            <id>estebanwasinger</id>
```

Comment:
isn't this one responsible of instagram's account rather than developing in mule?

Reply:
both things 📸

Reply:
follow @mulesoftarg

Reply:
Garcia Marra -> Gamarra

Reply:
☝️ , `Gamarra` FTW.